kubernetes/mariadb/skip-networking.cnf

```diff
@@ -1 +1,2 @@
-skip-networking
\ No newline at end of file
+[mysqld]
+skip-networking
```

Comment:
Please, consider applying this change also on the production manifests (https://github.com/kubic-project/caasp-container-manifests)